client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -74,99 +76,121 @@ const ProfileInfoEdit = () => {
-    toast.error('Profile has not been deleted');
+  const checkLogin = () => {
+    const accessToken = localStorage.getItem('GH_access_token');
+    if (accessToken === null) {
+      setLogInAlert(true);
+    }
+  };
+  const setDelete = (flag) => {
+    deleteFlag = flag;
   };
 
   return (
-    <div className="mt-5">
-      <Container className="profile-container">
-        <Form
-          className="profile-form"
-          noValidate
-          validated={validated}
-          onSubmit={handleUpdate}
-        >
-          <p className="profile-title-name">Team Profile Management</p>
-
-          <Form.Group controlId="formProfilePic">
-            <Image
-              src={profileData.profilePic}
-              roundedCircle
-              height="184px"
-              width="184px"
-            />
-            <Form.Control name="profilePic" type="file" accept="image/*" onChange={handleImageUpload} multiple={false} />
-            <Form.Text className="text-muted">
-              Upload a file from your device, at least 184px.
-            </Form.Text>
-          </Form.Group>
-
-          <Form.Group>
-            <Form.Label>Research Group Name</Form.Label>
-            <Form.Control
-              className="placeholder-text"
-              type="text"
-              placeholder="Allan Lab"
-              defaultValue={profileData.teamName}
-              onChange={updateInputs}
-              required
-              name="teamName"
-            />
-          </Form.Group>
-
-          <Form.Group>
-            <Form.Label>Organisation Name</Form.Label>
-            <Form.Control
-              className="placeholder-text"
-              type="text"
-              placeholder="Leiden University"
-              defaultValue={profileData.orgName}
-              onChange={updateInputs}
-              required
-              name="orgName"
-            />
-          </Form.Group>
-
-          <Form.Group>
-            <Form.Label>Email</Form.Label>
-            <Form.Control
-              className="placeholder-text"
-              type="email"
-              placeholder="allenlab@gmail.com"
-              name="email"
-              defaultValue={profileData.email}
-              onChange={updateInputs}
-              required
-            />
-          </Form.Group>
-
-          <div className="my-1">
-            <Button
-              id="updateButton"
-              type="submit"
-              color="primary"
-              className="mr-2"
-            >
-              Update
-            </Button>
-
-            {/* Button is linked to react-router-dom Link */}
-            <Link to="/dashboard">
-              <Button color="primary">Back</Button>
-            </Link>
-          </div>
-          <div className="my-1">
-            <Button variant="danger" onClick={profileDeleted}>
-              Delete account
-            </Button>
-          </div>
-        </Form>
-      </Container>
-    </div>
+    <>
```

Comment:
What is this empty html tag for?

Reply:
So I mainly followed what we had before and from what I understand it is a JSX fragment.  And when I remove it I get this error :
 Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>?